sim/midas/src/main/scala/midas/platform/VitisShim.scala

```diff
@@ -21,6 +21,9 @@ object VitisConstants {
 
   // This is wider than the addresses used in FPGATop
   val axi4MAddressBits = 64
+
+  /** The TCL variable name used to specify the simulator's frequecy */
```

Comment:
```suggestion
  /** The hardcoded TCL variable name used to specify the simulator's frequency */
```

Nit: Just making this explicit that this name is determined by the TCL. Then again, it is in the `constants` object so up 2 you on keeping this.